code/datums/diseases/advance/symptoms/fever.dm

```diff
@@ -36,6 +36,6 @@ Bonus
 	return
 
 /datum/symptom/fever/proc/Heat(mob/living/M, datum/disease/advance/A)
-	var/get_heat = (sqrt(21+A.totalTransmittable()*2))+(sqrt(20+A.totalStageSpeed()*3))
+	var/get_heat = (sqrt(max(0,21+A.totalTransmittable()*2)))+(sqrt(max(0,20+A.totalStageSpeed()*3)))

```

Comment:
are you sure the bug here isn't that one of `A.totalTransmittable` or `A.totalStageSpeed` is negative? This looks to me like you're just making it fail silently.
At the very least, it looks like you should make it `max(21,21+A.totalTransmittable()*2)` and `max(20,20+A.totalStageSpeed()*3)`